test/svg-sprite.test.js

```diff
@@ -213,6 +220,30 @@ describe('svg-sprite', () => {
                 });
             });
         });
+
+        describe(`with ${weather.length} relative path SVG files`, () => {
+            const spriter = new SVGSpriter({
+                shape: {
+                    dest: 'svg'
+                }
+            });
+
+            it(`returns ${weather.length} optimized shapes`, function(done) {
+                this.timeout(20_000);
```

Comment:
why do you add timeout?

Reply:
It could have been needed in the past, I removed it to see if CI is green